modules/component/types/componentProps.ts

```diff
@@ -0,0 +1,23 @@
+import {Action} from '@action-land/core'
+
+/**
+ * Component typing signature
+ * 1. **iState**: Initial state requiered by component to exist.
+ * 2. **oState**: Extends iState may contain additional properties.
```

Comment:
- extends iState already means `may contain additional properties`.
- if you want to keep it for clarity, put an `i.e.` in between them